src/run/docker-compose.ts

```diff
@@ -22,6 +22,8 @@ const CONTAINER_STOPPED_STATUS_PATTERN =
 const CONTAINER_STARTED_HEALTHY_STATUS_PATTERN =
     /(?:Container\s)?([\dA-Za-z-]+)\s*(Started|Healthy|Stopped|Pulling|Pulled)/;
 
+const AWS_PROFILE = process.env.AWS_PROFILE || "undefined";
```

Comment:
If I understand this correctly, this will set AWS_PROFILE to "undefined" if the environment variable does not exist...

Reply:
Do we really need to set it as a string? It will be `undefined` if not set, you could set `development-eu-west-2` as a default profile - if they've setup their machine using dev-env-setup then this profile should have been setup on their machine

Reply:
Wanted to avoid hardcoding the value incase there are changes to the profile in future, but if you prefer I can set it in the code.For better user experience, it shows an error message of 'undefined' when the profile is not set. 
Sample Error Message:

`Unable to retrieve credentials: The config profile (undefined) could not be found`

> Error: Fetch AWS credentials failed: Error: Command failed:
    aws configure export-credentials --profile undefined
    --format env

    Unable to retrieve credentials: The config profile
    (undefined) could not be found

    . Run: 'chs-dev troubleshoot analyse' command to
    troubleshoot.

Reply:
I think relying on aws CLI is not robust - if someone had a profile `undefined` then this error would not occur so is it safer just throw an error if `AWS_PROFILE` is undefined?

Reply:
Good point. Best to throw an error if AWS_PROFILE is undefined. We can be certain none of CH profiles will be called undefined(probably not). And if a user had a profile `undefined` it is definitely not what we want anyways.

Reply:
I think we don't need the default option - best just leave undefined and then handle the undefined later.

Reply:
Done.